packages/@aws-cdk/aws-stepfunctions-tasks/lib/athena/start-query-execution.ts

```diff
@@ -0,0 +1,259 @@
+          'glue:BatchDeleteTable',
+          'glue:BatchGetPartition',
+          'glue:CreateDatabase',
+          'glue:CreatePartition',
+          'glue:CreateTable',
+          'glue:DeleteDatabase',
+          'glue:DeletePartition',
+          'glue:DeleteTable',
+          'glue:GetDatabase',
+          'glue:GetDatabases',
+          'glue:GetPartition',
+          'glue:GetPartitions',
+          'glue:GetTable',
+          'glue:GetTables',
+          'glue:UpdateDatabase',
+          'glue:UpdatePartition',
+          'glue:UpdateTable'],
+        resources: [
+          cdk.Stack.of(this).formatArn({
+            service: 'glue',
+            resource: 'catalog',
+          }),
+          cdk.Stack.of(this).formatArn({
+            service: 'glue',
+            resource: 'database',
+            resourceName: this.props.queryExecutionContext?.database ? this.props.queryExecutionContext?.database : 'default',
+          }),
+          cdk.Stack.of(this).formatArn({
+            service: 'glue',
+            resource: 'table',
+            resourceName: '*', // access needed to query all tables https://docs.aws.amazon.com/athena/latest/ug/security-iam-athena.html
+          }),
+          cdk.Stack.of(this).formatArn({
+            service: 'glue',
+            resource: 'userdefinedfunction',
+            resourceName: '*', // users need access to start user defined functions from step functions https://docs.aws.amazon.com/athena/latest/ug/security-iam-athena.html
+          }),
+        ],
+      }),
+    );
+
+    return policyStatements;
+  }
+
+  /**
+   * Provides the Athena start query execution service integration task configuration
+   */
+  /**
+   * @internal
+   */
+  protected _renderTask(): any {
+    return {
+      Resource: integrationResourceArn('athena', 'startQueryExecution', this.integrationPattern),
+      Parameters: sfn.FieldUtils.renderObject({
+        QueryString: this.props.queryString,
+        ClientRequestToken: this.props.clientRequestToken,
+        QueryExecutionContext: {
+          Catalog: this.props.queryExecutionContext?.catalog,
+          Database: this.props.queryExecutionContext?.database,
+        },
+        ResultConfiguration: {
+          EncryptionConfiguration: {
+            EncryptionOption: this.props.resultConfiguration?.encryptionConfiguration?.encryptionOption,
+            KmsKey: this.props.resultConfiguration?.encryptionConfiguration?.kmsKey,
+          },
+          OutputLocation: this.props.resultConfiguration?.outputLocation,
+        },
+        WorkGroup: this.props.workGroup,
+      }),
+    };
+  }
+}
+
+/**
+ * Location of query result along with S3 bucket configuration
+ *
+ * @see https://docs.aws.amazon.com/athena/latest/APIReference/API_ResultConfiguration.html
+ * @experimental
+ */
+export interface ResultConfiguration {
+
+  /**ResultConfigurationOptions
+   * S3 path of query results
+   *
+   * @default - No output location
```

Comment:
where do query results go by default? are they just lost?
might also be handy to add `s3Location` in the name of the property. (there's a few examples of this in the repo)

Reply:
When one starts using Athena, one of the setup processes is under Settings to enter a Query Result Location for example: s3://query-results-bucket/folder/. This would be the default location but the user must create the bucket and key and can name it whatever they like.

Reply:
got it, maybe we can just describe that default instead